iotc-generic-c-sdk/include/iotconnect.h

```diff
@@ -43,14 +43,14 @@ typedef void (*IotConnectStatusCallback)(IotConnectConnectionStatus data);
 
 typedef struct {
     IotConnectAuthType type;
-    char* trust_store; // Path to a file containing the trust certificates for the remote MQTT host
+    const char* trust_store; // Path to a file containing the trust certificates for the remote MQTT host
```

Comment:
If the changes proposed to iotconnect_sdk_init_and_alloc_config() are done, we should revert back to char \* instead of const char \*